Whiskey/Functions/Install-WhiskeyTool.ps1

```diff
@@ -89,7 +89,7 @@ function Install-WhiskeyTool
     $startedWaitingAt = Get-Date
     $startedUsingAt = Get-Date
     $installLock = New-Object 'Threading.Mutex' $false,$mutexName
-    $DebugPreference = 'Continue'
+    #$DebugPreference = 'Continue'

```

Comment:
There is another uncommented `$DebugPreference = 'Continue'` at the bottom of this file in the `finally` block.

Reply:
Fixed.